src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+using Microsoft.Maui.Controls;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// DrawingView Platform Control
+/// </summary>
+public partial class MauiDrawingView
+{
+	readonly WeakEventManager weakEventManager = new();
+
+	/// <summary>
+	/// Event raised when drawing line completed 
+	/// </summary>
+	public event EventHandler<MauiDrawingLineCompletedEventArgs> DrawingLineCompleted
```

Comment:
so this on Core, not sure if we want to influence implementors to use an `Event`

Reply:
do you think we can remove it at all? I tried to keep it close to the IDrawingView

Reply:
Give me more time to think about this

Reply:
we need it. it is used in Handler. Assume it is a platform control like Android Button, which has event Clicked.